pytaxonkit.py

```diff
@@ -341,7 +341,7 @@ def lineage(
     >>> result[["TaxID", "Lineage", "LineageTaxIDs"]]
          TaxID                                                                Lineage                         LineageTaxIDs
     0  1325911    Eukaryota;Arthropoda;Insecta;Hymenoptera;Eucharitidae;Pogonocharis;  2759;6656;50557;7399;216140;1325911;
-    1  1649473  Bacteria;Bacteroidota;Cytophagia;Cytophagales;Spirosomaceae;Nibrella;  2;976;768503;768507;2896860;1649473;
+    1  1649473  Bacteria;Bacteroidota;Cytophagia;Cytophagales;Spirosomataceae;Nibrella;  2;976;768503;768507;2896860;1649473;
```

Comment:
This is fixing a test that was failing for me with the updated taxonomy database.